index.html

```diff
@@ -251,6 +251,12 @@
                         <div>
                             <div class="floe-front-news">
                                 <h3>News</h3>
+
+                                <h4>Inclusive Science Lab</h4>
+                                <p class="floe-date">July 19, 2017</p>
+                                <p>We've been working recently on designs to make science labs more inclusive. We recently invited two students to try out some things that we have built and to make a video with us.</p>
```

Comment:
I think this opening could be massaged a bit to read a bit better. Maybe:
"We've been working on designs to make science labs more inclusive, and invited two students to try some things we built. They also helped us make a video!"